index.bs

```diff
@@ -8426,10 +8426,10 @@ script.ArrayLocalValue = {
   value: script.ListLocalValue,
 }
 
-script.DateLocalValue = {
+script.DateLocalValue = (
```

Comment:
I think DateLocalValue is used in a type choice (/) in LocalValue and it looks like making it a group would not be quite correct?

Reply:
Alternatively we could use `{ script.DateLocalValue }` in the definition of `script.LocalValue`.

Reply:
By reading the CDDL RFC I am not sure it is even allowed to use type choice operator on groups. Our current version of the parser fails to parse this PR.

Reply:
That is correct. I am not aware of the limitation that groups can't be used in a type choice.

Reply:
Testing with something like: 

```
A = (
  a: int
)

B = {
  b: int
}

Root = A / B
```

does not work while 

```
A = {
  a: int
}

B = {
  b: int
}

Root = A / B
``` 

validates correctly.

Reply:
I think what Alex is referring to is the mix of groups and maps, which is not allowed.

Reply:
Yes, that's what I meant.

> Type operators (such as "/" for type choice)
>   operate on types, while group operators (such as "//" for group
>  choice) operate on groups.  Types can simply be used in groups, but
> groups need to be bracketed (as arrays or maps) to become types

https://datatracker.ietf.org/doc/html/rfc8610#section-3.11

I think we need to fix this before we land this PR (can it be as easy as wrapping groups into `{}` in the LocalValue definition like @whimboo suggested). I think removal of the `.and` struct is correct.

Reply:
> I think what Alex is referring to is the mix of groups and maps, which is not allowed.

@whimboo Yeah, I realized that after the fact, and just didn't delete my comment. Sorry for the wasted cognitive cycles.